config/crd/patches/webhook_in_developerusers.yaml

```diff
@@ -0,0 +1,17 @@
+# The following patch enables conversion webhook for CRD
+# CRD conversion requires k8s 1.13 or later.
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
The APIVersion i for `webhook_*` files and `cainjection_*` files should be `v1`

Reply:
fixed